pkg/users/account-details.js

```
@@ -109,7 +109,7 @@ export function AccountDetails({ accounts, groups, current_user, user, shells })
     const [edited_locked, set_edited_locked] = useState(null);
 
     function change_real_name() {
-        if (!edited_real_name)
+        if (edited_real_name == null || edited_real_name == undefined)
```

I think this probably goes in the right direction.

However, wouldn't we want `===` instead of `==`, as JavaScript should have the extra `=` in comparisons? Otherwise types can be silently converted via implicit type conversion.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

---

Correct. It might have gone over my head. I will make the necessary changes.